### DISCUSSION TITLE
I just enabled discussions

### DISCUSSION THREAD

**Discussion by phlogistonjohn** [ceph/go-ceph#443] [General]
Hi there. I wonder who will see this. Say hello by replying here!

**Comment by nixpanic** [comment]
On Wed, Jan 20, 2021 at 06:42:37AM -0800, John Mulligan wrote:
> Hi there. I wonder who will see this. Say hello by replying here!

Hello! If you read this, replying by email works ;-)

**Reply by phlogistonjohn** [reply]
Excellent. Github finally kinda/sorta supports mailing lists  ;-)

**Reply by ansiwen** [reply]
Ok, and there are threads as well...

**Comment by ansiwen** [comment]
This also comes via mail... :-)

John Mulligan <notifications@github.com> schrieb am Mi. 20. Jan. 2021 um
17:08:

**Comment by ansiwen** [comment]
> Hi there. I wonder who will see this. Say hello by replying here!

And this via iOS GitHub client...
